lps-client-backend/src/main/kotlin/no/nav/helsearbeidsgiver/lps/LpsClient.kt

```diff
@@ -17,7 +18,7 @@ private const val LPS_API_ENDPOINT = "https://sykepenger-im-lps-api.ekstern.dev.
 class LpsClient(
     var maskinportenService: MaskinportenService,
 ) {
-    suspend fun hentInntektsmeldinger(consumerOrgNr: String): List<InnsendtInntektsmelding> {
+    suspend fun hentInntektsmeldinger(consumerOrgNr: String): List<JsonObject> {
```

Comment:
Hva er hensikten bak dette byttet? Å videresende generiske responser?

Reply:
Egentlig bare at den gamle modellen var såpass utdatert, så felter og feltnavn stemte ikke. 
Dette er jo bare en POC for oss selv, så jeg så ikke helt poenget med å ta inn hele modellen / beskrivelsen fra vår spec inn i Tigersys når vi bare skal vise fram noen få felter - og litt mer fleksibelt, men kan absolutt lage et respons-objekt igjen etterhvert.